ibm/service/power/resource_ibm_pi_network_test.go

```diff
@@ -88,6 +88,43 @@ func TestAccIBMPINetworkGatewaybasic(t *testing.T) {
 		},
 	})
 }
+
```

Comment:
It was ok to update the existing basic test case. Just a suggestion... will let you decide, and mark it as resolved if you are good.

Reply:
Please mark it as a future enhancement. I am currently out and will be out until next Tuesday.